pypif/interop/mdf.py

```diff
@@ -0,0 +1,94 @@
+from pypif.util.read_view import ReadView
+from pypif.obj.common import Value
+from json import dumps
+
+
+def pif_to_mdf_record(pif):
+   """Convert a PIF into partial MDF record"""
+   res = {}
+   res["mdf"] = _to_meta_data(pif)
+   res["{source_name}"] = _to_user_defined(pif)
+   return dumps(res)
+
+
+def _to_meta_data(pif_obj):
+    """Convert the meta-data from the PIF into MDF"""
+    pif = pif_obj.as_dictionary()
+    mdf = {}
+    if pif["category"] == "system.chemical":
+        mdf["title"] = pif["names"][0]
```

Comment:
The names list might be (and is commonly) empty.  Could you fall back to chemical formula?